ext/src/Cassandra/Timeuuid.c

```diff
@@ -147,12 +157,12 @@ php_cassandra_timeuuid_properties(zval *object TSRMLS_DC)
 
   char* string = emalloc((CASS_UUID_STRING_LENGTH) * sizeof(char));
```

Comment:
Is `string` or `self->uuid` freed when calling `efree(self)` in [`php_cassandra_timeuuid_free`](https://github.com/datastax/php-driver/pull/58/files#diff-48d0a9b7bb2a2633ad51f5b3e165e6ebR207). Just checking in case we have a memory leak.